core-web/libs/dotcms-models/src/lib/dot-experiments.model.ts

```diff
@@ -183,32 +183,35 @@ export const ConditionDefaultByTypeOfGoal: Partial<Record<GOAL_TYPES, GOAL_PARAM
     [GOAL_TYPES.CLICK_ON_ELEMENT]: GOAL_PARAMETERS.URL
 };
 
+//Todo: Update the missing one with the new colors
 export const ChartColors = {
+    // Variants colors
+    default: {
+        line: 'rgb(66,194,240)',
+        fill: 'rgba(66,194,240,0.15)'
+    },
     primary: {
-        rgb: 'rgb(66,107,240)',
-        rgba_10: 'rgba(66,107,240,0.1)'
+        fill: getComputedStyle(document.body).getPropertyValue('--color-palette-primary-op-10'),
+        line: getComputedStyle(document.body).getPropertyValue('--color-palette-primary-op-70')
     },
     secondary: {
-        rgb: 'rgb(177,117,255)',
-        rgba_10: 'rgba(177,117,255,0.1)'
-    },
-    accent: {
-        rgb: 'rgb(65,219,247)',
-        rgba_10: 'rgba(65,219,247,0.1)'
+        fill: getComputedStyle(document.body).getPropertyValue('--color-palette-secondary-op-10'),
+        line: getComputedStyle(document.body).getPropertyValue('--color-palette-secondary-op-70')
     },
-    xAxis: { gridLine: '#AFB3C0' },
-    yAxis: { gridLine: '#3D404D' },
-    ticks: {
-        hex: '#524E5C'
+    // Chart colors
+    xAxis: {
+        border: getComputedStyle(document.body).getPropertyValue('--color-palette-black-op-20'),
+        gridLine: getComputedStyle(document.body).getPropertyValue('--color-palette-black-op-30')
     },
-    gridXLine: {
-        hex: '#AFB3C0'
+    yAxis: {
+        border: getComputedStyle(document.body).getPropertyValue('--color-palette-black-op-20'),
+        gridLine: getComputedStyle(document.body).getPropertyValue('--color-palette-black-op-50')
     },
-    gridYLine: {
-        hex: '#3D404D'
+    ticks: {
+        color: getComputedStyle(document.body).getPropertyValue('--color-palette-black-op-70')
```

Comment:
What if we create a centralize version of all this constants in JS, like:

```
export const themeColors = {
    //...
    colorPaletteBlackOp70: getComputedStyle(document.body).getPropertyValue('--color-palette-black-op-70')
}
```

I think it can be another ticket, what do you think @fmontes?

Reply:
Looking at the definition it takes an string... are you sure that you can't do `var(--color-palette-black-op-70)`?

Reply:
I tried. Not work in JS. I will centralize all like mention Jalinson.